File: Spring/les/src/main/java/com/jga/les/service/FornecedorService.java
package com.jga.les.service;

import com.jga.les.model.Fornecedor;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Service;

@Service
public class FornecedorService extends GenericService<Fornecedor> {

    public FornecedorService(JpaRepository<Fornecedor, Long> objRepository) {
        super(objRepository, Fornecedor.class);
    }
}

File: Flutter/les/lib/services/fornecedor_service.dart

import 'package:dio/dio.dart';
import 'package:les/core/generic_service.dart';
import 'package:les/model/fornecedor.dart';

class FornecedorService extends GenericService<Fornecedor> {

  FornecedorService(Dio dio) : super(dio, 'fornecedor', (data) => Fornecedor.fromJson(data));

}